docs/sphinx/examples/cpp/basics/cuquantum_backends.cpp

```diff
@@ -28,7 +28,7 @@ struct ghz {
 };
 
 int main() {
-  auto counts = cudaq::sample(ghz{}, 28);
+  auto counts = cudaq::sample(/*shots*/ 100, ghz{}, 28);
```

Comment:
```suggestion
  auto counts = cudaq::sample(/*shots=*/ 100, ghz{}, 28);
```
nit: silly LLVM style thing.